sway/desktop/render.c

```diff
@@ -54,6 +54,10 @@ bool should_parameters_blur() {
 	return config->blur_params.radius > 0 && config->blur_params.num_passes > 0;
 }
 
+int get_blur_size() {
+	return pow(2, config->blur_params.num_passes) * config->blur_params.radius;
```

Comment:
We should change this to `pow(2, config->blur_params.num_passes + 1) * config->blur_params.radius`. Seems like we didn't expand the damage enough in #120 which resulted in a tiny bit of the vignette showing:
![image](https://github.com/WillPower3309/swayfx/assets/35975961/e2405aae-a94d-453e-85d3-38dfb82f37b8)

Reply:
Would you prefer to change this in a separate PR to get 0.3 out of the door as soon as possible?